core/snippets/snippets/import.snippet

```diff
@@ -0,0 +1,14 @@
+name: importFromStatement
+phrase: import from
+
+---
+
+language: javascript | typescript | javascriptreact | typescriptreact
+-
+import $0 from "$1";
```

Comment:
Note that the original snippet from @AndreasArvidsson has `import $0 from "$0";`, I'm not sure what the intent was (my only guess is that `import foo from 'foo'` is somewhat common with npm libs).
This one is more generally applicable, and consistent with Python (where `from foo import foo` is quite rare).

`$1` is set to the name of the import location in both cases, as that allows using suggestions when you got to `$0`.